File: UI/bookbeauty_mobile/lib/models/appointment.dart
import 'package:json_annotation/json_annotation.dart';

part 'appointment.g.dart';

@JsonSerializable()
class Appointment {
  int? appointmentId;
  DateTime? dateTime;
  int? userId;
  int? hairdresserId;
  int? serviceId;
  String? note;
  Appointment(
      {this.appointmentId,
      this.dateTime,
      this.userId,
      this.hairdresserId,
      this.serviceId,
      this.note});

  factory Appointment.fromJson(Map<String, dynamic> json) =>
      _$AppointmentFromJson(json);

  Map<String, dynamic> toJson() => _$AppointmentToJson(this);
}

File: bookBeauty.Model/Appointment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace bookBeauty.Model
{
    public class Appointment
    {
        public int AppointmentId { get; set; }

        public DateTime? DateTime { get; set; }

        public int? UserId { get; set; }

        public int HairDresserId { get; set; } 

        public int ServiceId { get; set; }
    }
}
